PusherServer/ITriggerResult.cs

```diff
@@ -11,5 +11,10 @@ public interface ITriggerResult: IRequestResult
         /// Gets the Event IDs related to this Trigger Event
         /// </summary>
         IDictionary<string, string> EventIds { get; }
+
+        /// <summary>
+        /// If requested via trigger options, returns channel attributes for each channel in Trigger Event request
+        /// </summary>
+        IDictionary<string, ChannelAttributes> ChannelAttributes { get; }
```

Comment:
This is technically a Breaking Change. Since this appear in the return type, I think the unique workarround would be to create new methods in IPusher/Pusher. 

If you have any idea about how to do that without adding a BC, please, let me know.